src/Storage/DBFile.php

```diff
@@ -113,7 +113,9 @@ protected function getStore()
         'MimeType' => 'Varchar',
         'String' => 'Text',
         'Tag' => 'HTMLFragment',
-        'Size' => 'Varchar'
+        'Size' => 'Varchar',
+        'AttributesHTML' => 'HTMLFragment',
+        'getAttributesHTML' => 'HTMLFragment',
```

Comment:
do we need to separately specify getAttributesHTML? we don't specify getTag, we only have Tag

Reply:
That bit got copied over from `FormField`. You can still technically call `$getAttributesHTML` in your SS template.

Arguably we should have a `getTag` entry as well. Don't think it makes much difference either way. Don't think it makes much difference either way. I expect the vast majority of devs will use the method without the "get".